internal/web/templates/pages/support.html

```diff
@@ -1,40 +1,86 @@
+          target="_blank">kubeconfig</a>
+        file in the default location!
+      </p>
+      {{else if not .BootstrapMissing}}
+      <p class="text-center">Unfortunately, there is an error with your kubeconfig:</p>
+      <div class="alert alert-danger mb-3 font-monospace">{{.}}</div>
+      {{ end }}
 
-  {{ else if not .BootstrapMissing }}
-  <div class="col-md-6 p-lg-4 mx-auto">
-    <p class="text-center">
-      Unfortunately, there is an error with your kube config:
-    </p>
-    <p class="text-center lead mb-3 font-monospace">{{ .KubeconfigError }}</p>
-    <hr>
-    <small>
-      If you want to test Glasskube locally, check out <a class="text-dark" href="https://minikube.sigs.k8s.io/" target="_blank">minikube</a>.<br>
-      If you have any questions or encounter bugs, don't hesitate to reach out via <a href="https://github.com/glasskube/glasskube" class="text-dark" target="_blank">Github</a>
-      or <a href="https://discord.gg/SxH6KUCGH7" class="text-dark" target="_blank">Discord</a>.
-    </small>
-  </div>
+      Please get a kubeconfig file from your cloud provider and drag & drop it here:
+
+      <form method="post" enctype="multipart/form-data" action="/kubeconfig" class="d-flex flex-column gap-1 my-3">
+        <div id="dropzone" class="rounded border border-primary d-flex align-items-stretch" ondrop="dropHandler(event)"
+          ondragover="dragHandler(event)">
+          <label for="kubeconfig" style="cursor: pointer;" class="flex-grow-1 py-4 px-2 text-center">
+            <strong class="text-primary">Select a file</strong>
+            or drag & drop it here&hellip;
+          </label>
+          <input id="kubeconfig" name="kubeconfig" type="file" required class="visually-hidden"
+            onchange="changeHandler()">
+        </div>
+        <button id="submit-kubeconfig" type="submit" class="cta visually-hidden"></button>
+      </form>
 
-  {{ end }}
+      <p>Alternatively, you can copy it to this location on your computer:</p>
+      <p class="text-center lead mb-3 font-monospace">{{ .KubeconfigDefaultLocation }}</p>
+      <p>
+        Afterwards, stop the Glasskube Server in your terminal with CTRL+C, and then restart it with
+        <code>glasskube serve</code>.
+      </p>
+
+      <hr>
+
+      <small>
+        If you want to test Glasskube locally, check out
+        <a class="text-dark" href="https://minikube.sigs.k8s.io/" target="_blank">minikube</a>.<br>
+        If you have any questions or encounter bugs, don't hesitate to reach out via
+        <a href="https://github.com/glasskube/glasskube" class="text-dark" target="_blank">Github</a>
+        or <a href="https://discord.gg/SxH6KUCGH7" class="text-dark" target="_blank">Discord</a>.
+      </small>
+    </div>
+  </div>
 </div>
+
+<script type="text/javascript">
+  /**
+   * @type {HTMLInputElement}
+   */
+  const kubeconfigInput = document.getElementById('kubeconfig');
```

Comment:
During testing I encountered a bug (that I now can't reproduce anymore), but I think it was related to this line. When you upload an invalid kubeconfig, then "Go Back" to the support page, the file upload does not work anymore. Browser console says "Uncaught SyntaxError: Identifier 'kubeconfigInput' has already been declared".
Unfortunately I don't know exactly enough how I got to that point.

Reply:
Ok I think it was because I went back with the browser back button instead of the "Go back" link, in that case the upload doesn't work anymore (however without the browser console error). I think that's a marginal issue, the case of an invalid kubeconfig file will not happend that often and if so, people probably press the "Go back" button.

Reply:
Interesting. I think the problem could be that the JS is executed twice due to HTMX swapping the page instead of a full page reload. I'll check and see what I can do.

Reply:
I am unable to reproduce this bug in both Chromium and Firefox :confused: